VKapp/VKapp/Services/NetworkService/NetworkService.swift

```diff
@@ -82,16 +82,24 @@ final class NetworkService {
             }
     }
 
-    func fetchNews(urlString: String, completion: @escaping (Result<NewsFeedResponse, Error>) -> Void) {
-        AF.request(urlString).responseJSON { response in
-            guard let data = response.data else { return }
-            do {
-                let object = try JSONDecoder().decode(NewsFeedResult.self, from: data)
-                completion(.success(object.response))
-            } catch {
-                completion(.failure(error))
+    func fetchNews(
+        urlString: String,
+        nextPage: String? = "",
+        startTime: TimeInterval? = nil,
+        completion: @escaping (Result<NewsFeedResponse, Error>) -> Void
+    ) {
+        guard let nextPage = nextPage else { return }
+        let xx = urlString + "\(Api.startTimeText)\(startTime ?? 0)\(Api.startFromText)\(nextPage)"
```

Comment:
```suggestion
        let xx = urlString + "\(Api.startTimeText)\(startTime ?? 0)\(Api.startFromText)\(nextPage)"
```
Прикольное название xx :)